modules/apps/message-boards/message-boards-service/src/main/java/com/liferay/message/boards/service/impl/MBThreadLocalServiceImpl.java

```diff
@@ -58,13 +59,13 @@
 import com.liferay.portal.kernel.search.SearchContext;
 import com.liferay.portal.kernel.search.Sort;
 import com.liferay.portal.kernel.service.ServiceContext;
+import com.liferay.portal.kernel.service.SubscriptionLocalService;
 import com.liferay.portal.kernel.social.SocialActivityManagerUtil;
 import com.liferay.portal.kernel.systemevent.SystemEvent;
 import com.liferay.portal.kernel.util.StringUtil;
 import com.liferay.portal.kernel.util.Validator;
 import com.liferay.portal.kernel.workflow.WorkflowConstants;
 import com.liferay.social.kernel.model.SocialActivityConstants;
-import com.liferay.subscription.service.SubscriptionLocalService;
```

Comment:
Why did you change this???

Reply:
I ran source formatter and that did it. I just committed with the fix.